lib/tasks/gettext.rake

```diff
@@ -17,9 +17,18 @@ namespace :gettext do
   end
 
   def files_to_translate
-    GettextSetup.config['source_files'].map do |p|
+    files = GettextSetup.config['source_files'].map do |p|
       Dir.glob(p)
     end.flatten
+    # check for optional list of files to exclude from string
+    # extraction
+    if files_to_exclude = GettextSetup.config['exclude_files']
+      exclusions = files_to_exclude.map do |p|
+        Dir.glob(p)
+      end.flatten
+      return files - exclusions
```

Comment:
I'd like to see the `if` refactored out here. Something like:
```
exclusions = (GettextSetup.config['exclude_files'] || []).map do |p|
  Dir.glob(p)
end.flatten
files - exclusions
```
Having the single execution flow is stylistically better, I think.